### ISSUE TITLE
Order of Games Created

### ISSUE THREAD

**Issue by sx349** [wind23/whole_history_rating#10]
I was working on a WHR rating for Table Tennis players with ITTF match data. I noticed that when I input the match results in a rather random order, the ratings were quite unstable, e.g., there was a sudden dip in some player A's rating despite that player A did not lose (but instead he won a match) on that date. Such anomaly disappeared once I sorted the matches chronologically. 

From my understanding of the WHR algorithm, it should be immune to the order of games created if the games are taking into calculation simultaneously. But I assume that the actual algorithm works on an incremental basis and thus the order of games matters. Am I understanding it correctly? If so, we may want to warn the users to create games in a proper order.

**Comment by wind23** [comment]
You are right that the order in which games are created will affect the result of the algorithm. It may be a bug in the algorithm's implementation. I can check it later.
